lib/boat_tracker/sentences/parser.ex

```diff
@@ -0,0 +1,99 @@
+defmodule BoatTracker.Sentences.Parser do
+  @moduledoc """
+  NMEA sentences parser module.
+  """
+  alias BoatTracker.Sentences.RMC
+
+  @rmc_content_size 11
+  @time_size 6
+  @date_size 6
+
+  @spec parse(String.t()) :: RMC.t() | nil
```

Comment:
I think the documentation from `Sentences.RMC` would be more helpful here since this is where we're plucking off specific data points from the incoming sentence.